SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -2104,8 +2104,8 @@ void USpatialReceiver::ProcessQueuedActorRPCsOnEntityCreation(AActor* Actor, RPC
 	{
 		UFunction* Function = Info.RPCs[RPC.Index];
 		const FRPCInfo& RPCInfo = ClassInfoManager->GetRPCInfo(Actor, Function);
-		const FUnrealObjectRef ObjectRef = PackageMap->GetUnrealObjectRefFromObject(Actor);
-		check(ObjectRef != FUnrealObjectRef::UNRESOLVED_OBJECT_REF);
+		const FUnrealObjectRef ObjectRef(PackageMap->GetEntityIdFromObject(Actor), RPC.Offset);
+		check(ObjectRef.Entity != SpatialConstants::INVALID_ENTITY_ID);
```

Comment:
Why the change from unresolved?

Reply:
The equality operator for FUnrealObjectRef requires the offset to also be the same